Source/Project64-core/Project64-core.vcxproj

```diff
@@ -71,13 +71,13 @@
     <ClCompile Include="N64System\Recompiler\RecompilerOps.cpp" />
     <ClCompile Include="N64System\Recompiler\RegInfo.cpp" />
     <ClCompile Include="N64System\Recompiler\SectionInfo.cpp" />
-    <ClCompile Include="N64System\Recompiler\x86CodeLog.cpp" />
+    <ClCompile Include="N64System\Recompiler\X86CodeLog.cpp" />
     <ClCompile Include="N64System\Recompiler\X86ops.cpp" />
     <ClCompile Include="N64System\SpeedLimitorClass.cpp" />
     <ClCompile Include="N64System\SystemGlobals.cpp" />
     <ClCompile Include="Plugins\AudioPlugin.cpp" />
     <ClCompile Include="Plugins\ControllerPlugin.cpp" />
-    <ClCompile Include="Plugins\GFXplugin.cpp" />
+    <ClCompile Include="Plugins\GFXPlugin.cpp" />
```

Comment:
In the future, I'll just rename the PR as "<....> AND fix filename casing".
Good catch, though. I believe the actual issue is the inconsistent naming (the surce file is named GFXplugin, while the header is named GFXPlugin. The second seems more correct to me).